GoalSpeedAnywhere/GoalSpeedAnywhere.cpp

```diff
@@ -53,6 +53,10 @@ void GoalSpeedAnywhere::GetSpeed()
 	if(ball.IsNull()) return;
 	
     Speed = ball.GetVelocity().magnitude();
+    if(abs(ball.GetLocation().Y) > 5228.45)
```

Comment:
Wouldn't this check show the goal speed at very odd times in non-standard maps or workshop scenarios?

Reply:
That's a good point. It was copied from the disable freeplay goal feature in BM and was thrown in here as sort of a hack in the hopes it would fix things. Since it doesn't seem to have done the trick (and as you said it would cause more issues in non-standard maps) I'll revert the merge.